src/website/app/demos/Tabs/shared/content.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+export default function(
+  index: number | string,
+  { rtl }: { rtl: boolean } = {}
+) {
+  return rtl
+    ? `${index} خدمة الجزر أبجد هوز. شرب الحالي سمبر مصنوعة من السكر. الغاز وكالات التصنيف الائتماني في تأثير تتعهد. اللاعبين التمويل الدردشة عن العقارات الكلي. الواجبات المنزلية الآن أن شرائح حزينة الكيميائية البيئية. قبل التربة خوفا من زين، وعظيم الكرتون الرئيسي. الشيخوخة محزن والمرض والجوع والتغذية سكان الفقر القبيح. في الواقع، في قطر، في معرف بوروس، لا الاحماء. منتصف الجلوس تلقي الكازينو. أسد، والكثير من كرة القدم. هذا العنصر هو أن البوابة الرئيسية. ومع ذلك، فإن الموز التطوير العقاري قوس الصلصة.`
+    : `${index} Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
I really think that it would be an improvement to have more significant content differences across tabs.  The tiny little number change is not very noticeable and differently sized content could help illustrate additional aspects, e.g. the need to potentially add a fixed height to prevent jumpiness.